src/function/scalar/generic/current_setting.cpp

```diff
@@ -24,6 +26,17 @@ static void CurrentSettingFunction(DataChunk &args, ExpressionState &state, Vect
 	result.Reference(info.value);
 }
 
+static string NormalizeSettingName(const string &name) {
+	auto lower = StringUtil::Lower(name);
```

Comment:
Same here, explicit `StringUtil::Lower` should not be required if we are using the `case_insensitive_map`.